### ISSUE TITLE
[2.5.2] Build Errors on macOS High Sierra

### ISSUE THREAD

**Issue by Varunram** [Musicoin/go-musicoin#70]
@5chdn I was building 2.5.2 on macOS High Sierra 10.13.2 and I got the following errors:

```
>>> /usr/local/Cellar/go/1.9.2/libexec/bin/go install -ldflags -X main.gitCommit=cce00784ba746d05cdd7a60ffb964df97b3b6513 -s -v ./cmd/gmc
github.com/ethereum/go-ethereum/params
# github.com/ethereum/go-ethereum/params
params/config.go:91:80: cannot use false (type bool) as type *big.Int in field value
params/config.go:91:97: cannot use big.NewInt(0) (type *big.Int) as type bool in field value
params/config.go:91:113: cannot use common.Hash literal (type common.Hash) as type *big.Int in field value
params/config.go:91:127: cannot use big.NewInt(0) (type *big.Int) as type common.Hash in field value
params/config.go:91:165: cannot use new(EthashConfig) (type *EthashConfig) as type *big.Int in field value
params/config.go:91:181: too few values in struct initializer
params/config.go:98:80: cannot use false (type bool) as type *big.Int in field value
params/config.go:98:97: cannot use big.NewInt(0) (type *big.Int) as type bool in field value
params/config.go:98:113: cannot use common.Hash literal (type common.Hash) as type *big.Int in field value
params/config.go:98:127: cannot use big.NewInt(0) (type *big.Int) as type common.Hash in field value
params/config.go:98:127: too many errors
util.go:45: exit status 2
exit status 1
make: *** [gmc] Error 1
```
The standard build geth procedure was followed. Where am I going wrong?

**Comment by 5chdn** [comment]
#69 fixed this, c1b3fa066d8df993999d5bf2851890c273a77f80

**Comment by Varunram** [comment]
Ah, makes sense. I'm not too are about it, its been a while since I tested this one out. Thanks for the quick response!
